app/routes/pages/PageDetailRoute.js

```diff
@@ -109,11 +110,8 @@ const loadData = async (props, dispatch) => {
         .concat(dispatch(fetchAllPages()))
     );
   }
-  const itemActions = [];
+  const itemActions = dispatchInOrder(fetchItemActions, dispatch, pageSlug);
```

Comment:
In this case, do you really need to dispatch them in exact order? Can't all `fetchItem`s be called at the same time?

Reply:
I believe the committee actions have to be dispatched consecutively, as you need a group in the state in order to mutate it's membership.

Reply:
I believe the committee actions have to be dispatched consecutively, as you need a group in the state in order to mutate it's membership.

Reply:
I believe the committee actions have to be dispatched consecutively, as you need a group in the state in order to mutate it's membership.